### PR TITLE
Parser bond xmit hash policy

### PR DESCRIPTION
**Pull request by vishwanathjadhav**
Enhanced parser Bond to get "Transmit Hash Policy" filed.

Trello:
https://trello.com/c/hVReG4kK

### CODE REVIEW

File: insights/parsers/bond.py [hunk 1]
```diff
@@ -132,3 +140,11 @@ def aggregator_id(self):
         bond file, ``[]`` is returned.
         """
         return self._aggregator_id
+
+    @property
```
Comment (PaulWay) [inline insights/parsers/bond.py]: Please don't use property methods, just set the attribute directly.

File: insights/parsers/bond.py [hunk 2]
```diff
@@ -100,6 +104,10 @@ def parse_content(self, content):
                 self._slave_interface.append(line.split(":", 1)[1].strip())
             elif line.strip().startswith("Aggregator ID: "):
                 self._aggregator_id.append(line.strip().split(':', 1)[1].strip())
+            elif line.strip().startswith("Transmit Hash Policy"):
+                # No need of values in bracket:
+                # Intiger notification (0), (1), (2) of layer2, layer3+4, layer2+3 resp
+                self._xmit_hash_policy = line.split(":", 1)[1].split()[0]
```
Comment (PaulWay) [inline insights/parsers/bond.py]: Please just set 'self.xmit_hash_policy' here directly.

File: insights/parsers/bond.py [hunk 3]
```diff
@@ -100,6 +104,10 @@ def parse_content(self, content):
                 self._slave_interface.append(line.split(":", 1)[1].strip())
             elif line.strip().startswith("Aggregator ID: "):
                 self._aggregator_id.append(line.strip().split(':', 1)[1].strip())
+            elif line.strip().startswith("Transmit Hash Policy"):
+                # No need of values in bracket:
+                # Intiger notification (0), (1), (2) of layer2, layer3+4, layer2+3 resp
```
Comment (PaulWay) [inline insights/parsers/bond.py]: Minor spelling correction: 'intiger' -> 'integer'

File: insights/parsers/bond.py [hunk 4]
```diff
@@ -83,6 +86,7 @@ class Bond(Parser):
     def parse_content(self, content):
         self._bond_mode = None
         self._partner_mac_address = None
+        self.xmit_hash_policy = None
```
Comment (xiangce) [inline insights/parsers/bond.py]: Would you please add the new attribute to docstring?

Comment (vishwanathjadhav) [reply insights/parsers/bond.py]: added attribute string

**Pull request merged**